app/src/main/kotlin/com/wire/android/ui/common/Logo.kt

```diff
@@ -19,8 +20,8 @@ fun Logo() {
         contentDescription = stringResource(id = R.string.app_logo_description),
         contentScale = ContentScale.Fit,
         modifier = Modifier
-            .width(80.dp)
-            .height(30.dp)
-            .padding(8.dp)
+            .padding(paddingValues = PaddingValues(horizontal = 8.dp, vertical = 32.dp))
```

Comment:
is there a specific reason why you use PaddingValues ? you can just do it like this : padding(horizontal = 8.dp, vertical = 32.dp), also those .dp to dimensions file